### PR TITLE
feat(arch): move mvn to gradle

### LINKED ISSUE #19: move mvn to gradel
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by pelukron**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [x] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Related Tickets & Documents

- Related Issue #16 
- Closes #19 

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

## Added/updated tests?

- [x] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

### CODE REVIEW

**Comment by pelukron** [comment]
@ccs-ocampo this pr is auto merge, be careful when you give the approval

File: build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	}
+}
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	implementation("org.springframework.boot:spring-boot-starter")
+	compileOnly("org.projectlombok:lombok")
```
Comment (ccs-ocampo) [inline build.gradle.kts]: Why do you need this library?

File: build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		extendsFrom(configurations.annotationProcessor.get())
+	}
+}
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	implementation("org.springframework.boot:spring-boot-starter")
```
Comment (ccs-ocampo) [inline build.gradle.kts]: Why did you added this one?

File: build.gradle.kts (hunk 1 above)
Comment (pelukron) [reply build.gradle.kts]: refactored this file

File: build.gradle.kts (hunk 2 above)
Comment (pelukron) [reply build.gradle.kts]: refactored this file

**Pull request merged**